odbc-api/tests/integration.rs

```diff
@@ -2792,39 +2799,41 @@ fn list_tables_preallocated(profile: &Profile, expected: &str) {
     let cursor = preallocated
         .tables(None, None, Some(table_name), None)
         .unwrap();
-    let actual = cursor_to_string(cursor);
+    let actual = cursor_to_string(cursor).to_lowercase();
 
-    assert_eq!(expected, actual);
+    assert_eq!(expected.to_lowercase(), actual);
 }
 
-/// List columns for various data sources
-#[test_case(MSSQL, "master,dbo,ListColumns,a,4,int,10,4,0,10,1,NULL,NULL,4,NULL,NULL,2,YES,0,0,0,0,NULL,NULL,NULL,NULL,NULL,NULL,38"; "Microsoft SQL Server")]
-#[test_case(MARIADB, "test_db,NULL,ListColumns,a,4,INT,10,4,0,10,1,,NULL,4,NULL,2,2,YES"; "Maria DB")]
-#[test_case(SQLITE_3, ",,ListColumns,a,4,INTEGER,9,10,10,0,1,NULL,NULL,4,NULL,16384,2,YES"; "SQLite 3")]
-fn list_columns(profile: &Profile, expected: &str) {
-    let table_name = "ListColumns";
-    let conn = profile.setup_empty_table(table_name, &["INTEGER"]).unwrap();
+// TODO(Soremwar)
```

Comment:
That's interesting. I mostly develop locally on a windows machine, and sometimes on subsystem for linux. I never had any trouble with the memory consumption of this (or any other) test.

Reply:
It's not a matter of memory consumption, when requesting MariaDB for the type of a column it's calculating a size that's way too big despite the fact that's only a varchar

Reply:
Specifically, when SQLColAttributeW is asked to give a DisplaySize for it (data type is INT), it returns 536870911 instead of 64 as it should do

Any ideas?